docs/operation_center/rules_catalog.md

```diff
@@ -1,336 +1,441 @@
+
+```yaml
 detection:
   <Search-Idenfier>
     <string-list>
     <field:value>
   condition: <Condition>
-``` 
+```
+
+`<Search-Identifier>` is a unique identifier that will be used in `<Condition>`. A `<Search-Idenfier>` can contain two different structures: **Maps** and **Lists**.
+
+#### Maps
 
-`<Search-Identifier>` is a unique identifier that will be used in `<Condition>`. A `<Search-Idenfier>` can contain two different structures, lists and maps.
+Maps consist of key/value pairs in which the key is the name of a field in your normalized event. The value is the value that should be present in this field for the `Search-Identifier` to match. Several values can be specified at once, in which case they are joined together with a logical `OR`.
 
-!!! Note
-    Currently, we set a limitation and rules can be written for only one event. Aggregation expressions are not supported yet.
-    
-For the full specification of SIGMA, please refers to the [SIGMA Github](https://github.com/SigmaHQ/sigma/wiki/Specification)
+The keys of a map are joined together with a logical `AND`.
 
-### Lists
-A list contains one or more strings that will match the full event log. Multiple strings will be linked with a logicial `OR`.
+**Example** : Raise an alert when an event contains `event.type: allowed` and the `destination.ip` is either `1.2.3.4` or `4.3.2.1`
 
-**Example** : Raise an alert when an event contains "DangerousThreat.exe" or "powershell.exe malicousfile.ps1"
+```yaml
+detection:
+  selection:
+    event.type: allowed
+    destination.ip:
+      - 1.2.3.4
+      - 4.3.2.1
+  condition: selection
 ```
+
+#### Lists
+
+A list contains one or more items that are joined together with a logical `OR`.
+
+Lists of strings can be used to define any string that should be present in the whole event `message`:
+
+**Example**: Raise an alert when an event contains `DangerousThreat.exe` or `powershell.exe maliciousfile.ps1`
+
+```yaml
 detection:
   keywords:
     - DangerousThreat.exe
     - powershell.exe maliciousfile.ps1
   condition: keywords
 ```
 
-### Maps
-Maps consists of key/value pairs, in which the key is a field in ECS (Elastic Common Format). ECS is the format you can see in the `Details` tab in the `Events` page.
-Multiple key/value within the same map are linked with a logical `OR` while two different maps are linked with a logiciel `AND`.
+You can also use lists of maps to combine several maps with a logical `OR`.
 
-**Example** : Raise an alert when an event contains `event.type: allowed` and the destination IP is either `destination.ip: 1.2.3.4` or `destination.ip: 4.3.2.1`
-```
+**Example**: Raise an alert when a specific IP address is seen as a source or a destination:
+
+```yaml
 detection:
   selection:
-    event.type: allowed
-    destination.ip:
-    - 1.2.3.4
-    - 4.3.2.1
+    - source.ip: 6.6.6.6
+    - detection.ip: 6.6.6.6
```

Comment:
Hello,
Why have you chosen `detection.ip` and not `destination.ip:`  ? :thinking:

Reply:
Nice catch, thx :+1: